recipe/meta.yaml

```diff
@@ -175,6 +175,8 @@ requirements:
     - eigen 3.3.7
     - astunparse 1.6.3
     - opentelemetry-api
+  run:  # [win]
```

Comment:
this needs to be done for other plats too btw. satisfies overlinking check? Although we might want to understand the megabuild better to see if it's the right solution